src/OrbitVulkanLayer/VulkanLayerController.h

```diff
@@ -125,11 +143,31 @@ class VulkanLayerController {
     // Advance linkage for next layer
     layer_create_info->u.pLayerInfo = layer_create_info->u.pLayerInfo->pNext;
 
+    std::vector<std::string> all_extension_names{};
+    all_extension_names.assign(
```

Comment:
You added a lengthy explanation in the commit message, but I think it would be even more beneficial to add such explanation in one or more comments in the code.